src/test/java/BASE.java

```diff
@@ -2,26 +2,25 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
 import java.time.Duration;
-import java.util.List;
-import java.util.Set;
 
 public class BASE {
```

Comment:
By the way, with page class you can create folder pages and put the classes related to the page into that folder. 
example: BasePage, LoginPage, ..
-> this will help you maintain the code better